cypress/features/regression/designSystem/drawer.feature

```diff
@@ -44,16 +44,4 @@ Feature: Drawer component
   @positive
   Scenario: Set expandedWidth to 62%
     When I click on expanded-width-62-drawer Drawers arrow 1 time
```

Comment:
I think we need also remove this TC as we have tested the expanded width for Drawer using `chromatic`

Reply:
removed